back-end/API/app/services/user_service.py

```
@@ -118,4 +126,16 @@ def linkToExisting(self, offer, selectedEnterpriseId):
                 enterprise_repo.deleteEnterprise(duplicatedEnterprise.id)
                 employer_repo.linkEmployerEnterprise(user.id, selectedEnterpriseId)
             else:
-                raise Exception("trying to delete a non temporary enterprise.")
\ No newline at end of file
+                raise Exception("trying to delete a non temporary enterprise.")
+            
+    
+    def _generateToken(self, user):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_generateToken`

(ANN202)